File: Exemplos/ExemploClasse.py
class Jogador:

    def __init__(self, nome, time, apelido, posicao, camisa, salario):
        self.__nome = nome
        self.__time = time
        self.__apelido = apelido
        self.__posicao = posicao
        self.__camisa = camisa
        self.__salario = salario
        print("Jogador Criado")

    def calcularSalrioAnual (self):
        return self.__salario*12

File: Exemplos/ExemploClasse.java
public class ExemploClasse {
    
}

public class Jogador {
  
  String nome, time, apelido, posicao;
  float salario;
  int camisa;
    
  public Jogador() {

    System.out.println("Jogador criado");
        
  }
    
  public Jogador(String nome, String time, String apelido, String posicao, int camisa, float salario) {
 
    this.nome = nome;
    this.time = time;
    this.apelido = apelido;
    this.posicao = posicao;
    this.camisa = camisa;
    this.salario = salario;
    System.out.println("Jogador criado");
        
  }
  
  public float calcularSalarioAnual () {  
    return (float) (this.salario*12.00);
  }
  
}
    
